src/Mod/Cloud/Gui/PreCompiled.cpp

```diff
@@ -0,0 +1,24 @@
+/***************************************************************************
+ *   Copyright (c) 2019         jmverdun3@gmail.com	                   *

```

Comment:
please add you name before the email and fix whitespace after it

Reply:
Shall be fixed